src_c/circle.c

```diff
@@ -7,6 +7,8 @@
 #include <stddef.h>
 #include <math.h>
 
+#define PI (double)3.14159265358979323846264
+#define TAU (double)6.28318530717958647692528
```

Comment:
i would think that you dont need to cast to double here as firstly you dont have float literals and secondly the number is already long enough that it cant be a float